sdk/src/main/java/com/horizen/account/serialization/EthJsonMapper.java

```diff
@@ -0,0 +1,47 @@
+package com.horizen.account.serialization;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import com.horizen.evm.utils.BigIntegerDeserializer;
+import com.horizen.evm.utils.BigIntegerSerializer;
+
+import java.math.BigInteger;
+
+public class EthJsonMapper {
+
+    private static final ObjectMapper mapper;
+
+    static {
+        var module = new SimpleModule();
+        module.addSerializer(BigInteger.class, new BigIntegerSerializer());
+        module.addDeserializer(BigInteger.class, new BigIntegerDeserializer());
+        module.addSerializer(byte[].class, new EthByteSerializer());
+        module.addDeserializer(byte[].class, new EthByteDeserializer());
+        mapper = new ObjectMapper();
+        mapper.registerModule(module);
+        mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+        // do not serialize null or empty values
+//        mapper.setSerializationInclusion(JsonInclude.Include.NON_NULL);
+//        mapper.setSerializationInclusion(JsonInclude.Include.NON_EMPTY);
+//        mapper.setSerializationInclusion(JsonInclude.Include.NON_ABSENT);
+//        mapper.enable(SerializationFeature.INDENT_OUTPUT);
```

Comment:
how does it exactly work for us now?
`null` fields are serialized as `null`? and what about the deserialization?

Reply:
Yeah, I'm a bit unclear on how to do this properly. We can not just omit every `null`, because at least in a transaction we must include the `"to":null`. Maybe we can omit `null`, but overwrite the behavior for `tx.to`.

Reply:
ok, clear.
I think we may keep only `mapper.enable(SerializationFeature.INDENT_OUTPUT);`. It helps to read the outputs during the debug, for example.

other commented configs can be removed. We need to serialize `null`s by default and where it is not needed -> add annotation, like `@JsonInclude(Include.NON_NULL)` or `@JsonInclude(Include.NON_EMPTY)`